config/economicsConfig.go

```diff
@@ -8,17 +8,16 @@ type EconomicsAddresses struct {
 
 // RewardsSettings will hold economics rewards settings
 type RewardsSettings struct {
-	RewardsValue        uint64
+	RewardsValue        string
 	CommunityPercentage float64
 	LeaderPercentage    float64
 	BurnPercentage      float64
 }
 
 // FeeSettings will hold economics fee settings
 type FeeSettings struct {
-	MinGasPrice      uint64
-	MinGasLimitForTx uint64
-	MinTxFee         uint64
+	MinGasPrice      string
+	MinGasLimitForTx string
```

Comment:
There will be no minimum gas for smart contract. You can try to send with the same gasLimit, but it will be not enough. You cannot estimate and set a config for it. Delete ForTx.

Reply:
Done.